007-AzureMonitoring/Coach/Solution-02.md

```diff
@@ -0,0 +1,72 @@
+# Challenge 02 - Setting up Monitoring via Automation - Coach's Guide 
+
+[< Previous Solution](./Solution-01.md) - **[Home](./README.md)** - [Next Solution >](./Solution-03.md)
+
+## Notes & Guidance
+
+### Deploy alerts with Bicep
+
+The Action group Resource Id can be found in the Azure Portal:
+- In Azure Monitor navigate to Alerts, then Action groups.
+- Select the Action group to open its properties.
+- Click on JSON View.
+
+![Navigating to JSON view of the action group in Azure Portal](../Images/02-01-Find-resource-id.png)
+
+- At the top you will see the Resource ID that will have the following structure:  
+`/subscriptions/<sub-id>/resourceGroups/<rg-name>/providers/microsoft.insights/actionGroups/<action-group-name> `
+  
+
+### Test Activity log alert
+
+- Login to Azure portal and stop you Visual Studio VM by clicking the "Stop" button.  
+
+![Stopping the  Virtual Machine via “Stop” button in Azure Portal](https://github.com/msghaleb/AzureMonitorHackathon/raw/master/images/stopVM.png)
```

Comment:
Commenting here but I see them in a number of solution files.  Will the links to the images be changed to the local files?  I see some pointing to local images and others pointing to another repo.

Reply:
This needs to be fixed/udpated!

Reply:
Done!